### ISSUE TITLE
Language client is not ready yet

### ISSUE THREAD

**Issue by woter1832** [znck/grammarly#394]
VSCode has been open for hours.

I've removed and reinstalled the plugin.

<img width="971" alt="image" src="https://github.com/znck/grammarly/assets/68354513/4fca2639-9df4-41c9-bac1-eee6cfe8030a">

Same as #379 although I'm on Windows 11.

No VPN or firewall.

I believe this output is from Grammarly:

```
Ready!
Initialized!
open file:///c%3A/Users/<user>/source/<company>/RCZ/rccc/rcz-docs/docs/enterprise-scale/organisational-parameters/naming-convention.md
create text checking session for "file:///c%3A/Users/<user>/source/<company>/RCZ/rccc/rcz-docs/docs/enterprise-scale/organisational-parameters/naming-convention.md" with {
  "documentDialect": "auto-text",
  "documentDomain": "general",
  "suggestions": {
    "ConjunctionAtStartOfSentence": false,
    "Fluency": true,
    "InformalPronounsAcademic": false,
    "MissingSpaces": true,
    "NounStrings": true,
    "NumbersBeginningSentences": true,
    "NumbersZeroThroughTen": true,
    "OxfordComma": false,
    "PassiveVoice": false,
    "PersonFirstLanguage": true,
    "PossiblyBiasedLanguageAgeRelated": true,
    "PossiblyBiasedLanguageDisabilityRelated": true,
    "PossiblyBiasedLanguageFamilyRelated": true,
    "PossiblyBiasedLanguageGenderRelated": true,
    "PossiblyBiasedLanguageHumanRights": true,
    "PossiblyBiasedLanguageHumanRightsRelated": true,
    "PossiblyBiasedLanguageLgbtqiaRelated": true,
    "PossiblyBiasedLanguageRaceEthnicityRelated": true,
    "PossiblyPoliticallyIncorrectLanguage": true,
    "PrepositionAtTheEndOfSentence": false,
    "PunctuationWithQuotation": true,
    "ReadabilityFillerwords": true,
    "ReadabilityTransforms": true,
    "SentenceVariety": true,
    "SpacesSurroundingSlash": true,
    "SplitInfinitive": true,
    "StylisticFragments": false,
    "UnnecessaryEllipses": false,
    "Variety": true,
    "Vocabulary": true
  }
} 
node:internal/deps/undici/undici:11413
    Error.captureStackTrace(err, this);
          ^

TypeError: fetch failed
    at Object.fetch (node:internal/deps/undici/undici:11413:11) {
  cause: Error: unknown scheme
      at makeNetworkError (node:internal/deps/undici/undici:6768:35)
      at schemeFetch (node:internal/deps/undici/undici:10875:18)
      at node:internal/deps/undici/undici:10748:26
      at mainFetch (node:internal/deps/undici/undici:10765:11)
      at fetching (node:internal/deps/undici/undici:10722:7)
      at fetch2 (node:internal/deps/undici/undici:10600:20)
      at Object.fetch (node:internal/deps/undici/undici:11411:18)
      at fetch (node:internal/process/pre_execution:248:25)
      at c:\Users\<user>\.vscode\extensions\znck.grammarly-0.22.1\dist\server\index.node.js:34:77333
      at c:\Users\<user>\.vscode\extensions\znck.grammarly-0.22.1\dist\server\index.node.js:34:77553 {
    [cause]: undefined
  }
}

Node.js v18.15.0
[Error - 2:30:32 AM] Connection to server got closed. Server will not be restarted.

```

**Comment by AlfredLarsen** [comment]
Same error on MacOS. Error appeared a few days ago. Extension currently not working.

**Comment by kulttuuri** [comment]
Same error. Change in Grammarly API or API key expired?

**Comment by znck** [comment]
This was caused by web-tree-sitter failing to initialize on Node 18. Fixed today in v0.24.0
